common/changes/@typespec/compiler/fix-f5-validate_2023-06-23-16-47.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@typespec/compiler",
+      "comment": "",
```

Comment:
No comment here because regression was intra-sprint and only impacts TypeSpec team development not customers at that.